test/test_optim.py

```diff
@@ -113,6 +123,9 @@ def _test_basic_cases_template(self, weight, bias, input, constructor):
         input = Variable(input)
         optimizer = constructor(weight, bias)
 
+        # to check if the optimizer can be printed as a string
+        optimizer.__repr__()
```

Comment:
It's enough to call this here and test it once. Please remove all other places

Reply:
Thank you. This has been done as suggested.